code/_helpers/unsorted.dm

```diff
@@ -35,6 +35,15 @@
 	return text("#[][][]", textr, textg, textb)
 	return
 
+/proc/hex2rgba(var/hex, var/alpha)

```

Comment:
This is considered a "type2type" conversion, so it'd go in the `type2type.dm` file. Tho maybe this should also be named `hex2htmlrgba` or something, since it doesn't exactly return an rgb object. 🤔